api/bundles.class.php

```diff
@@ -0,0 +1,138 @@
+<?php
+  require_once('util.php'); // for version_compare_*
+
+  class Bundles {
+    function get_bundles($filter, $keyboards, $versionInfo, $get_keyboard_info) {
+      $result = [];
+      $versionInfo = json_decode($versionInfo);
+      $appver = $versionInfo->windows->stable;
```

Comment:
I see `$appver` is the latest Desktop stable version. Can you clarify how `$appver` is different from very similarly named `$appVersion` below?